plugins/openvr/src/OpenVrHelpers.cpp

```diff
@@ -122,6 +124,14 @@ vr::IVRSystem* acquireOpenVrSystem() {
             #if DEV_BUILD
                 qCDebug(displayplugins) << "OpenVR display: HMD is " << activeHmd << " error is " << eError;
             #endif
+
+            if (eError == 108) { // vr::HmdError_Init_HmdNotFound
```

Comment:
108 is a magic number,  please just use `vr::VRInitError_Init_HmdNotFound` instead and remove the comment

Reply:
I couldn't it kept erroring :c

Reply:
note that the constant you were using is different than the one I showed you `vr::HmdError_Init_HmdNotFound`  != `vr::VRInitError_Init_HmdNotFound`     perhaps the docs you were looking at were from another API version?

Reply:
That is always possible, Eitherway, will do it when VCPKG finishes taking an eternity